geometry_msgs/Point pose
sensor_msgs/Image image